kubernetes/samples/charts/traefik/README.md

```diff
@@ -91,18 +91,18 @@ TLS_PORT=`kubectl -n traefik get service traefik-operator -o jsonpath='{.spec.po
 $ curl -k -H 'host: domain1.org' https://${HOSTNAME}:${TLS_PORT}/testwebapp/
 ```
 
-## SSL Termination at Ingress Controller
-This sample demonstrates how to terminate SSL trafik at ingress contoller to access WebLogic console thru SSL port. 
+## SSL termination at ingress controller
+This sample demonstrates how to terminate SSL traffic at ingress  to access the WebLogic Server Administration Console through SSL port. 
```

Comment:
at ingress  to access -> ??? at the ingress controller to access
through SSL port -> through the SSL port

Reply:
done